### PR TITLE
[UI] changes the color of  "X" from red to the theme color of UI 

### PR DESCRIPTION
**Pull request by Chadha93**
Signed-off-by: Gaurav Chadha <gauravchadha1676@gmail.com>

**Description**
changes the color of  "X"  on hover from red to the theme color of UI in :

- Filters
- Patterns
- Applications

This PR fixes # Style

**Notes for Reviewers**
We don't have any issues with red but it's not in our game. :P 

https://user-images.githubusercontent.com/65453826/135730186-eb6882dd-4a14-4deb-9214-ac5baae96530.mp4

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by warunicorn19** [comment]
@Chadha93 some screenshots would be great😅.

**Comment by Chadha93** [comment]
> @Chadha93 some screenshots would be greatsweat_smile.

Sure adding with few more commits

**Comment by Chadha93** [comment]
> @Chadha93 some screenshots would be greatsweat_smile.

Added @warunicorn19

**Review by warunicorn19 — DISMISSED** [review]
LGTM, you're right the red isn't looking that good😅

**Review by warunicorn19 — APPROVED** [review]

**Review by leecalcote — APPROVED** [review]

File: ui/components/Filters.js [hunk 1]
```diff
@@ -165,6 +166,43 @@ function MesheryFilters({
...
+            borderBottom : "2px solid #222"
+          },
+          "&:after" : {
+            borderBottom : "2px solid #222"
+          }
+        }
+      },
+      MUIDataTableSearch : {
+        searchIcon : {
+          color : "#607d8b" ,
```
Comment (leecalcote) [inline ui/components/Filters.js:183]: @Tanuj22 we have this color in the central theme, right?

**Review by leecalcote — APPROVED** [review]

**Review by leecalcote — CHANGES_REQUESTED** [review]
Reuse elements from central theme, please.

**Comment by warunicorn19** [comment]
@leecalcote I don't think we have a central theme yet🤔.

File: ui/components/Filters.js (hunk 1 above)
Comment (Chadha93) [reply ui/components/Filters.js:183]: @leecalcote I used the same central(#codes) that we have all across.

**Comment by Chadha93** [comment]
> Reuse elements from central theme, please.

We don't have defined central theming yet ig. // @sudo-NithishKarthik

**Pull request merged**